sdk/src/Services/S3/Custom/Transfer/_async/TransferUtility.async.cs

```diff
@@ -208,11 +210,15 @@ public partial class TransferUtility : ITransferUtility
         ///     A cancellation token that can be used by other objects or threads to receive notice of cancellation.
         /// </param>
         /// <returns>The task object representing the asynchronous operation.</returns>
-        public Task UploadAsync(TransferUtilityUploadRequest request, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task UploadAsync(TransferUtilityUploadRequest request, CancellationToken cancellationToken = default(CancellationToken))
         {
-            CheckForBlockedArn(request.BucketName, "Upload");
-            var command = GetUploadCommand(request, null);
-            return command.ExecuteAsync(cancellationToken);
+            using (var span = AWSConfigs.TelemetryProvider.TracerProvider.GetTracer(S3TransferTracerScope)
```

Comment:
We should be using the client specific `TelemetryProvider`, since we allow the user to override the provider at the service client level
```
this._s3Client.Config.TelemetryProvider.TracerProvider
```

Reply:
Done